easybuild/easyconfigs/g/GenomeWorks/GenomeWorks-2021.02.2-fosscuda-2020b.eb

```diff
@@ -0,0 +1,135 @@
+# author: Denis Kristak (INUITS)
+easyblock = 'CMakePythonPackage'
+
+name = 'GenomeWorks'
+version = '2021.02.2'
+local_git_commit = '691380d1122a9ad86f91b3038c8b6625d137b6ed'
+
+homepage = 'https://github.com/clara-parabricks/GenomeWorks'
+description = "SDK for GPU accelerated genome assembly and analysis"
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/clara-parabricks/GenomeWorks/archive/'],
+        'filename': 'v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/google/benchmark/archive'],
+        'filename': 'benchmark-20190513.tar.gz',
+        'download_filename': '090faecb454fbd6e6e17a75ef8146acb037118d4.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/NVIDIA/cub/archive'],
+        'filename': 'cub-20201123.tar.gz',
+        'download_filename': '618a46c27764f0e0b86fb3643a572ed039180ad8.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/google/googletest/archive'],
+        'filename': 'googletest-20190424.tar.gz',
+        'download_filename': '84d986531e8df9fb6ea4a7bae8fb130c05b3782c.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/samtools/htslib/archive'],
+        'filename': 'htslib-20200806.tar.gz',
+        'download_filename': 'a79009b38ce83e39bcbc8f54c00cf203621aa5bb.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/cartoonist/kseqpp/archive'],
+        'filename': 'kseqpp-20181009.tar.gz',
+        'download_filename': 'c7add979ac77ec8ada07b25aab7dcf0c5e87f90e.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/NVIDIA/libcudacxx/archive'],
+        'filename': 'libcudacxx-20201003.tar.gz',
+        'download_filename': 'd0bafdb9fdde2ee99932d42f1deec74d53e19514.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/spoa/archive'],
+        'filename': 'spoa-20190416.tar.gz',
+        'download_filename': '0b9e56da3241d6a887d64adcfd1ebfae29810c04.tar.gz',
+    },
+]
+checksums = [
+    'f036f066b3ae22e603786f79d374cde28942c22064a169f123fb085fd6b4ccaf',  # v2021.02.2.tar.gz
```

Comment:
If you use `git_config`, use `checksums = [None]`, since the source tarball is created locally on download then, and the checksum won't be identical across different systems...